instrumentation/kamon-akka/src/common/resources/reference.conf

```diff
@@ -192,4 +193,18 @@ kanela.modules {
       "akka.serialization..*"
     ]
   }
-}
\ No newline at end of file
+
+  actor-monitor {
+    name = "Kamon actor monitor instrumentation"
+    description = "Extracts correct message class for akka 2.6.x"
+    enabled = yes
+
+    instrumentations = [
+      "kamon.instrumentation.akka.instrumentations.akka_26.ActorMonitorInstrumentation"
+    ]
+
+    within = [
+      "kamon.instrumentation.akka.instrumentations..*"
+    ]
+  }
```

Comment:
Please move this instrumentation to the `akka` module. It doesn't make sense to have it as a separate module if we don't want to enable/disable independently (in fact, this one should never be disabled!) and if we have it as an independent module it will show up in the Status Page as well, probably confusing users a bit.